.github/workflows/enforce-conventional-commits.yml

```
@@ -0,0 +1,14 @@
+name: Conventional Commits
+
+on:
+  pull_request:
+    branches: [ development ]
```

```suggestion
    branches: * 
```

I'm not sure if this is the correct syntax but we should enforce on every branch because what if a bounty hunter has a main branch for their PR and then sub branches with the wrong commit styles. Wouldn't those commits be difficult to parse for any automated tooling down the line?

---

Currently the merge flow is the following:
1. Bounty hunters create PRs to the `development` branch
2. You manually merge `development` into `main` from time to time

There is no way to miss invalid commit msg because all PRs are getting in the `development` branch first

---

I would much rather have the bounty hunters handle the squash merge / fix commit message issue than me!